src/Sulu/Bundle/ContactBundle/Resources/public/js/services/contact-manager.js

```diff
@@ -64,7 +64,7 @@ define([
             if (!!data.categories) {
                 contact.get('categories').reset();
                 Util.foreach(data.categories, function(id) {
-                    var category = Category.findOrCreate({id: id});
+                    var category = Category.findOrCreate({id: (!id.id ? id : id.id)});
```

Comment:
@wachterjohannes I am going to revert this change. Doesn't look like this change has a consequence on this feature, and it looks to me like there is a wrong call else where, and this issue should be fixed there, not here.